tests/js/app/testBuffetHistoryStore.js

```diff
@@ -1,163 +0,0 @@
-            model: model,
-        });
-        expect(store.get_current_item().page_type).toBe(Pages.SET);
-    });
-
-    function test_close_lightbox (action, descriptor) {
-        it('closes the lightbox when ' + descriptor, function () {
-            let model = new DModel.Article();
-            let media_model = new DModel.Media();
-            store.set_current_item_from_props({
-                page_type: Pages.ARTICLE,
-                model: model,
-                media_model: media_model,
-            });
-            dispatcher.dispatch({
-                action_type: action,
-            });
-            expect(store.get_current_item().media_model).toBeNull();
-        });
-    }
-    test_close_lightbox(Actions.LIGHTBOX_CLOSED, 'lightbox close clicked');
-    test_close_lightbox(Actions.SEARCH_BOX_FOCUSED, 'search box focused');
-
-    function test_search_action (action, descriptor) {
-        describe('when ' + descriptor, function () {
-            beforeEach(function () {
-                dispatcher.dispatch({
-                    action_type: action,
-                    search_terms: 'foo',
-                });
-            });
-
-            it('goes to the search page', function () {
-                expect(store.get_current_item().page_type).toBe(Pages.SEARCH);
-            });
-
-            it('records a metric', function () {
-                expect(AppUtils.record_search_metric).toHaveBeenCalled();
-            });
-        });
-    }
-    test_search_action(Actions.SEARCH_TEXT_ENTERED, 'search text entered');
-    test_search_action(Actions.DBUS_LOAD_QUERY_CALLED, 'desktop search opened');
-
-    describe('when link in article clicked', function () {
-        it('goes to article page', function () {
-            let model = new DModel.Article();
-            engine.get_object_promise.and.returnValue(Promise.resolve(model));
-            dispatcher.dispatch({
-                action_type: Actions.ARTICLE_LINK_CLICKED,
-                ekn_id: 'ekn://foo/bar',
-            });
-            Utils.update_gui();
-            expect(store.get_current_item().page_type).toBe(Pages.ARTICLE);
-        });
-
-        it('shows lightbox if link was media', function () {
-            let model = new DModel.Media({
-                ekn_id: 'ekn://foo/pix',
-            });
-            engine.get_object_promise.and.returnValue(Promise.resolve(model));
-            dispatcher.dispatch({
-                action_type: Actions.ARTICLE_LINK_CLICKED,
-                ekn_id: 'ekn://foo/pix',
-            });
-            Utils.update_gui();
-            expect(store.get_current_item().media_model).toBe(model);
-        });
-    });
-
-    describe('when an article card is clicked', function () {
-        beforeEach(function () {
-            let model = new DModel.Article();
-
-            dispatcher.dispatch({
-                action_type: Actions.ITEM_CLICKED,
-                model: model,
-                context: [model],
```

Comment:
I would suggest adding these context parameters to the corresponding dispatch actions in testMeshHistoryStore.js.